src/test/java/org/terracotta/statistics/derived/histogram/BarSplittingBiasedHistogramPerformanceTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2015 Terracotta, Inc..
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.statistics.derived.histogram;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Random;
+
+@Ignore
```

Comment:
JMH?

Reply:
Probably should be... but I'm not massively bothered to change it right now.